libs/mep/nala/script.js

```diff
@@ -0,0 +1,3 @@
+// give a one-second delay to allow the DOM structure to exist
+
+setTimeout(() => { document.querySelector('body').style.color = 'orange'; }, 1000);
```

Comment:
This seems a bit flaky... depending on what you want to achieve AFTER milo finished rendering we add 
`<div id="page-load-ok-milo" style="display: none;"></div>` ... otherwise you could use an intersection observer

Reply:
It is a bit flaky, but it is only intended to test a feature of MEP called "insertScript". 
https://main--milo--adobecom.hlx.page/drafts/nala/features/personalization/mep-actions/insert-script

Reply:
I could make it more robust, if needed

Reply:
@mokimo I made the code more robust